src/Administration/Resources/app/administration/src/module/sw-plugin/view/sw-plugin-list/sw-plugin-list.html.twig

```diff
@@ -43,7 +43,8 @@
                             :showPreviews="true"
                             :sortBy="sortBy"
                             :sortDirection="sortDirection"
-                            :compactMode="false">
+                            :compactMode="false"
+                            @doSearch="getList">
```

Comment:
The `sw-entiity-listing` should support pagination out of the box. I think it is better to find out the source of the problem then to fix the symptoms :)